### PR TITLE
[ISSUE-3399][linkis-engineplugin-python]errorcode code optimization

### PR DESCRIPTION
**Pull request by binbinCheng**
**[linkis-engineplugin-python]errorcode code optimization**
Related issue:#https://github.com/apache/incubator-linkis/issues/3399

### CODE REVIEW

File: docs/errorcode/python-errorcode.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+## python errorcode
+
+| 模块名(服务名) | 错误码  | 描述 | Exception Class|
+| -------- | -------- | ----- |-----|
+|python|400201|Invalid python session.(无效的 python 会话.)", "Invalid python session.(无效的 python 会话.)|LinkisPythonErrorCodeSummary|
```
Comment (casionone) [inline docs/errorcode/python-errorcode.md:5]: duplicate, please remove

File: docs/errorcode/python-errorcode.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+|python|60003|Pyspark process  has stopped, query failed!(Pyspark 进程已停止，查询失败！)|LinkisPythonErrorCodeSummary|
+|python|60003| |LinkisPythonErrorCodeSummary|
```
Comment (casionone) [inline docs/errorcode/python-errorcode.md:7]: Incorrect please correct

Comment (casionone) [reply docs/errorcode/python-errorcode.md:7]: Add enum  name such as  INVALID_PYTHON_SESSION


| 模块名(服务名) | 错误码  |错误名| 描述 | Exception Class|
| -------- | -------- | ----- |-----|-----|
|python|400201|INVALID_PYTHON_SESSION|Invalid python session.(无效的 python 会话.)".)|LinkisPythonErrorCodeSummary|
|python|60003|PYSPARK_PROCESSS_STOPPED|Pyspark process  has stopped, query failed!(Pyspark 进程已停止，查询失败！)|LinkisPythonErrorCodeSummary|
|python|41001|PYTHON_EXECUTE_ERROR| |LinkisPythonErrorCodeSummary|

**Review by jacktao007 — APPROVED** [review]
LGTM.

**Pull request merged**